python/paddle/onnx/export.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from paddle.utils import try_import
+
+__all__ = ['export']
+
+
+def export(layer, path, input_spec=None, opset_version=9, **configs):
+    """
+    Export Layer as ONNX format model, which can be used for inference.
```

Comment:
Export Layer as -> Export Layer to

Reply:
which can be used for inference with onnxruntime.

Reply:
done